src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/ClientConfigPaths.cs

```diff
@@ -265,9 +264,12 @@ private static string GetTypeAndHashSuffix(string exePath, bool isSingleFile)
                 {
                     try
                     {
-                        // Certain platforms may not have support for crypto
-                        hash = IdentityHelper.GetNormalizedUriHash(codeBase);
-                        typeName = UrlDesc;
+                        if (Uri.TryCreate(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, assembly.ManifestModule.Name), UriKind.RelativeOrAbsolute, out Uri codeBase))
```

Comment:
The constructor that was used here before used `UriKind.Absolute` kind: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.Uri/src/System/Uri.cs#L365-L371

Is there a reason to switch to `UriKind.RelativeOrAbsolute` here?

Reply:
Good catch, was just trying to cover everything possible, did not checked the Uri implementation